lambda/src/main/scala/pricemigrationengine/model/Either.scala

```diff
@@ -0,0 +1,15 @@
+package pricemigrationengine.model
+
+object Either {
+
+  /**
+    * Converts a Seq of Eithers to either a Left
+    * containing the first Left value of the Seq if any of the Seq has a Left value,
+    * or else to a Right that contains the Seq of all the Right values in the original Seq.
+    */
+  def fromSeq[E, A](eithers: Seq[Either[E, A]]): Either[E, Seq[A]] =
```

Comment:
Are you trying to keep the lambda size small? For familiarity you could call it `sequence` and provide it as extension method

```
// to keep the lambda package small we are not importing cats
implicit final class TraverseOps[E, A](eithers: Seq[Either[E, A]]) {
  def sequence: Either[E, Seq[A]] = {
    eithers
      .collectFirst { case Left(e) => e }
      .map(Left(_))
      .getOrElse(Right(eithers.collect { case Right(a) => a }))
  }
}
```

so now you can use the familiar `l.sequence` syntax.

Reply:
That's a good idea.  I'll do it.  I didn't want to bring in a whole library for this although I had a feeling it was done elsewhere.